ext/SymbolicsGroebnerExt.jl

```diff
@@ -89,7 +89,7 @@ function Symbolics.solve_multivar(eqs::Vector, vars::Vector{Num}; dropmultiplici
         new_eqs = copy(eqs)
         eq = new_var
         for i = 1:(old_len)
-            eq += rand(n_iterations*-10:n_iterations*10)*vars[i]
+            eq += BigInt(rand(n_iterations*-10:n_iterations*10))*vars[i]
```

Comment:
Smaller coefficients may reduce the size of the coefficients of the separating form.
Say `big(rand(-n_iterations:n_iterations))`.
What do you think ?